dipy/tracking/streamlinespeed.pyx

```diff
@@ -102,15 +102,18 @@ def length(streamlines):
         arclengths = np.zeros(len(streamlines), dtype=np.float64)
 
         if streamlines.data.dtype == np.float32:
-            c_arclengths_from_arraysequence[float2d](streamlines.data,
-                                                     streamlines._offsets,
```

Comment:
@MarcCote - any thoughts here?  I guess these streamlines are somehow of type long, but they should be of type np.intp.  Any way of making them be np.intp from the start, to avoid memory costs of the conversion?